packages/@lwc/engine/src/framework/vm.ts

```diff
@@ -265,9 +251,40 @@ export function createVM(elm: HTMLElement, Ctor: ComponentConstructor, options:
     linkComponent(initializedVm);
 }
 
+function assertIsVM(obj: any): asserts obj is VM {
+    if (!isObject(obj) || !('cmpRoot' in obj)) {
+        throw new TypeError(`${obj} is not a VM.`);
+    }
+}
+
+export function associateVM(obj: VMAssociable, vm: VM): void {
```

Comment:
if it doesn't return anything, we normally (at until now), don't add the returned type declaration.